pyproject.toml

```diff
@@ -23,14 +23,9 @@ documentation = "https://docs.litestar.dev/"
 keywords = [
     "api",
     "rest",
-    "http",
```

Comment:
What's the reason for removing some of these keywords?

Reply:
linting errors in my IDE letting me know that keywords are limited to 5 max. intellij got updated.